app-config.yaml

```diff
@@ -2,8 +2,9 @@ httpApi:
   host: 0.0.0.0
   port: 3000
   # Secret keys for JWT
-  refreshTokenSecret: 'refreshTokenSecret',
-  accessTokenSecret: 'accessTokenSecret',
+  refreshTokenSecret: 'refreshTokenSecret'
+  accessTokenSecret: 'accessTokenSecret'
+  environment: 'development'
```

Comment:
are you sure it should be included at the `httpApi`? maybe at the root?

Reply:
done